lib/server.js

```diff
@@ -27,10 +27,7 @@ var Router = util.safeRequire('react-router');
 
 // a template of the `script` tag that gets
 // injected into the server rendered pages.
-var TEMPLATE = ['<script id="%s" type="application/javascript">var ',
-                Config.client.variableName,
-                ' = %s;</script>'
-              ].join('');
+var TEMPLATE = '<script id="%s" type="application/javascript">window._store = %s;</script>';
```

Comment:
Gotcha, I wasn't sure. The example you wrote in the comment contained `window._store` - thought you wanted to change the API to something more native to the browser. 

I'll revert back to the original definition.